central/views/imagecve/view_test.go

```diff
@@ -1,5 +1,3 @@
-//go:build sql_integration
```

Comment:
Need to put this back for your test I think.

Reply:
Ah yes!